src/components/TimelineNav/TimelineNav.stories.tsx

```diff
@@ -2,22 +2,22 @@ import { BADGE } from '@geometricpanda/storybook-addon-badges';
 import { StoryObj, Meta } from '@storybook/react';
```

Comment:
Could you add a `.test.tsx` file for this component that generates snapshots? I just realized we haven't been doing that when moving components over. Example file: https://github.com/chanzuckerberg/edu-design-system/blob/main/src/components/Banner/Banner.test.tsx

Reply:
Done. Thanks!